CHANGES.md

```diff
@@ -1,5 +1,13 @@
 # Change log for kotlinx.atomicfu
 
+# Version 0.15.0
+
+* Tracing atomic operations
```

Comment:
It's worth linking the corresponding issues/PR's as we do in coroutines: https://github.com/Kotlin/kotlinx.coroutines/blob/master/CHANGES.md

Reply:
Done